filer/templates/admin/filer/folder/directory_listing.html

```diff
@@ -130,7 +130,7 @@
                             <div class="navigator-breadcrumbs-folder-name-wrapper">
                                 <span class="navigator-breadcrumbs-folder-name">
                                     <span class="navigator-breadcrumbs-folder-name-inner">
-                                        {% if folder.name != "unfiled files" %}
+                                        {% if folder.name != "Unsorted Uploads" %}
```

Comment:
if you compare this and `_("Unsorted Uploads")`, it could return `false` if it's not English